### ISSUE TITLE
ERROR: The CommonCrypto.h header file could not be found. (XCode 7.2 - iOS 9.2)

### ISSUE THREAD

**Issue by rmvz3** [iosdevzone/IDZSwiftCommonCrypto#10]
I'm trying to add IDZSwiftCommonCrypto to my project via CocoPods bug I'm getting this error:

    ERROR: The CommonCrypto.h header file could not be found.

Is there anything I can do?

Thx

**Comment by tako3** [comment]
I have forked version of IDZSwiftCommonCrypto.
This might work :+1:

```
use_frameworks!

pod 'IDZSwiftCommonCrypto', :git => 'https://github.com/tako3/IDZSwiftCommonCrypto.git', :branch => 'develop'
```

**Comment by rmvz3** [comment]
Hey, thank you Tako3!

It works like a charm

**Comment by iosdevzone** [comment]
@rmvz3 sorry I could not respond to you earlier... I'm on vacation with limited internet access.
@tako3 thanks for solving this. 
I'll be back online on the 29th and will figure out a longer term fix for this problem.

**Comment by iosdevzone** [comment]
I have merged PR #11 from @tako3 (Thanks!) and tagged the release 0.6.8 (and pushed this to CocoaPods). This should solve this problem.
